fv/test-workload/test-workload.go

```diff
@@ -316,3 +337,20 @@ func writeProcSys(path, value string) error {
 	}
 	return err
 }
+
+func doIptablesSetup() error {
```

Comment:
Worth a comment that the purpose of this is to simulate a sidecar?

Reply:
Done. Also renamed it to `doSidecarIptablesSetup`.